libp2p-playground/README.md

```diff
@@ -0,0 +1,44 @@
+Test environment for debugging the interaction of multiple nodes via libp2p with a specified network topology.
```

Comment:
Couldn't we put it into a separate repository?

Looks a bit unrelated to nil project.

Reply:
I've been thinking about that option, too. I suppose we could do that. After all, it's never too late to move it back if it becomes necessary.

Reply:
My main concern is using of Docker and Redis. This is not typical for our repository where we use nix and focus only on nil development.